galaxykubeman/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: 21.01
 description: A chart for deploying and managing a single Galaxy out-of-the-box instance on Managed Kubernetes clusters (GKE)
 name: galaxykubeman
-version: 1.0.0
+version: 1.0.1
```

Comment:
Figured it's good to bump the patch version

Reply:
Since 1.0.0 doesn't really work and is not being consumed by anyone, I would argue we could just overwrite it since this is a bugfix, but also happy to bump too

Reply:
overwrite 1.0.0 is fine with me